Makefile

```diff
@@ -73,7 +73,7 @@ else
 			HOST_ARCH = x86
 		endif
 		ifeq ($(shell uname -m),armv7l)
-			HOST_ARCH = armv7l
+			HOST_ARCH = armhf
```

Comment:
Yeah, I thought about that, however I think it'd be even more confusing that various `armv7` similar architectures (like `armv7l` and `armv7hf`) generate the same `armhf` package. What do you think?

Since they all result in the same generic architecture, maybe its worth to convey that very clearly from the start?